pkgs/development/python-modules/panflute/default.nix

```diff
@@ -26,6 +27,11 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "panflute" ];
 
+  postPatch = ''
+    substituteInPlace panflute/tools.py \
+      --replace-fail "PANDOC_PATH = None" "PANDOC_PATH = '${pandoc}/bin/pandoc'"
+  '';
+
```

Comment:
Why do you want to provide pandoc at build time?

Reply:
should i use 
```py
  preCheck = ''
    export PATH="$PATH:${pandoc}/bin"
  '';
```
in `pandoc-latex-environment` instead?

Reply:
Just add `pandoc` to `nativeCheckInputs`.